charts/karpenter/README.md

```diff
@@ -66,6 +67,7 @@ helm upgrade --install --namespace karpenter --create-namespace \
 | strategy | object | `{"type":"Recreate"}` | Strategy for updating the pod. |
 | terminationGracePeriodSeconds | string | `nil` | Override the default termination grace period for the pod. |
 | tolerations | list | `[]` | Tolerations to allow the pod to be scheduled to nodes with taints. |
+| topologySpreadConstraints | list | `[]` | Constraints to spread replicas across the cluster. |
```

Comment:
Why can't you use `affinity`? Unless you're running an unbounded number of replicas this would be better practice.